R/backend-hana.R

```diff
@@ -20,7 +20,7 @@
 #' library(dplyr, warn.conflicts = FALSE)
 #'
 #' lf <- lazy_frame(a = TRUE, b = 1, c = 2, d = "z", con = simulate_hana())
-#' lf %>% transmute(x = paste0(z, " times"))
+#' lf %>% transmute(x = paste0(d, " times"))
```

Comment:
All of these minor fixes definitely imply that the old behaviour was silently obscuring a lot of problems.